lib/generators/listed/certificate_key_generator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails/generators'
+require 'rails/generators/migration'
+require 'rails/generators/active_record'
+
+require 'generators/lets_encrypt/register_generator'
+
+module Listed
+  module Generators
+    class CertificateKeyGenerator < LetsEncrypt::Generators::RegisterGenerator
+      private
+
+      def generate_key
+        # rubocop:disable Metrics/LineLength
+        key_path = ask("Where you to save private key [#{LetsEncrypt.private_key_path}]:", path: true)
```

Comment:
"Where do you want to save private key"?

Reply:
It's for generating the private key one time only for myself locally so I can inject it later into the env vars.

Originally it was this generator: https://github.com/elct9620/rails-letsencrypt/blob/master/lib/generators/lets_encrypt/register_generator.rb#L28

But I had to overwrite the method because it generated a 4096-bit RSA key and AWS accepts up to 2048 for now. So it's mostly copied for extending this one method with one-line difference.

Reply:
Ah ok. Was just commenting on the sentence itself, as it seems to be missing a few words ;)

Reply:
weird part is, I didn't notice that 😂